css/styling.css

```
@@ -155,7 +159,8 @@ header li a:hover, footer li a:hover  {
 }
 
 
-input[type="text"], input[type="number"], input[type="button"], input[type="password"] {
+input[type="text"], input[type="number"], input[type="button"], input[type="password"], input[type="tel"],
+input[type="email"], input[type="date"], input[type="time"] {
```

```suggestion
input{
```
kan man ikke gøre noget i den her stil?